contrib/ansible/tasks/bootstrap_ubuntu.yaml

```diff
@@ -0,0 +1,16 @@
+---
+- name: "Install required packages on Ubuntu"
+  package:
+    name: "{{ item }}"
+    state: latest
+  with_items:
+      - unzip
+      - tar
+      - apt-transport-https
+      - btrfs-tools
+      - libapparmor-dev
```

Comment:
TODO to figure out whether we need the dev libraries or what are the alternatives.